presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -191,6 +193,13 @@ else if (config.getPinGlueClientToCurrentRegion()) {
             }
         }
 
+        if (config.getIamRole().isPresent()) {
+            AWSCredentialsProvider credentialsProvider = new STSAssumeRoleSessionCredentialsProvider
+                    .Builder(config.getIamRole().get(), "presto-session")
```

Comment:
shall we have a `roleSessionName ` instead of using `presto-session`?

Reply:
addressed. you meant  ` roleSessionName ` string as session name instead of `presto-session`